src/NHibernate/Transaction/AdoNetWithDistributedTransactionFactory.cs

```diff
@@ -1,3 +1,4 @@
+#if !NETSTANDARD2_0
```

Comment:
Start reading from here: https://github.com/dotnet/corefx/issues/15046 => https://github.com/dotnet/standard/issues/157 => https://github.com/dotnet/standard/pull/168 

Basically it is supported in a standard, but not yet implemented everywhere